README.md

```diff
@@ -101,6 +93,64 @@ file.
 
 Note that new entry points will need the project to be unmounted and mounted again to be applied.
 
+### On the target machine (Wazo)
+
+#### Install dependencies
+Install rsync, and create a folder where repos must be available
+
+```sh
+apt update
+apt install rsync
+mkdir /usr/src/wazo  # or whatever your <local_source>
+```
+
+Install ssh
+```sh
+apt update
+apt install openssh-server
+```
+
+confiure ssh to enable root login
+```sh
+nano /etc/ssh/sshd_config
+```
+
+Add this line
+```sh
+PermitRootLogin yes
+```
+
+Restart sshd
+```sh
+sudo systemctl restart ssh
+```
+
+#### Cloning repos
+The repos you want to mount must be available in the local source folder
+
+```sh
+cd /usr/src/wazo
+git clone https://github.com/wazo-platform/the-repo-you-want
```

Comment:
```suggestion
wdk repo clone
```

The wazo repositories should be cloned to the configure local source directory(the `local_source` key in the config.yml, defaults to `~/wazo`).